src/main/java/securbank/dao/AccountDao.java

```diff
@@ -0,0 +1,20 @@
+package securbank.dao;
+
+import securbank.models.Account;
+
+import java.util.*;
+
+/**
+ * 
+ * @author Madhu Illuri
+ *
+ */
+
+public interface AccountDao extends BaseDao<Account, UUID> {
+	
+	public Account findByAccountNum(UUID accountNumber);
```

Comment:
@madhuilluri This function is already defined in BaseDao. We don't need to redefine it here.